pyproject.toml

```diff
@@ -30,8 +30,13 @@ httpx = "^0.18.2"
 hypothesis = "^6.14.0"
 black = "^21.6b0"
 watchgod = "^0.7"
-
+taskipy = "^1.9.0"
 
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.taskipy.tasks]
+start = "uvicorn api.main:app --host 0.0.0.0 --port 8000 --reload"
+lint = "black . && flake8 ."
```

Comment:
This should probably run `black --check` (or maybe `black --check --diff`) instead, as this is a check.